rfc/0032-field-level-encryption.md

```diff
@@ -728,16 +729,24 @@ There are a few items out of scope at this time for v1.0, however they very much
   * Algorithm upgrading - for example going from AES-256 to AES-512
   * Support for KMIP based key stores
 
+
+## Changelog
+
+YYYY-MM-DD Initial draft publication
+
+2019-10-31
+* Specified that the Base64 encoder must emit padding if applicable.
+* Removed conflicting/incorrect specification for signature calculation.
+
+
 ## Sign Off
 
 |Language |Representative |Date |
-|:--- |:--- |:--- |--- |
+|--- |--- |--- |
```

Comment:
Why are you removing the alignment specifiers?

Reply:
Because I am dumb and didn't know that's what they were. The important change on this line is to remove the extra  column, since the table data only has 3 columns, and putting 4 columns in the header caused the table to render incorrectly.